.github/workflows/tests.yml

```diff
@@ -51,7 +53,8 @@ jobs:
     strategy:
       matrix:
         features:
-          - "--features iouring"
+          - "--features commonware-runtime/iouring-storage"
+          - "--features commonware-runtime/iouring-network"
```

Comment:
Until network is passing reliably, I'm wondering if we want to leave a TODO to re-enable?

Could put TODO as #954